src/hotspot/share/opto/vectornode.hpp

```diff
@@ -93,7 +93,7 @@ class VectorNode : public TypeNode {
   static bool is_type_transition_short_to_int(Node* n);
   static bool is_type_transition_to_int(Node* n);
   static bool is_muladds2i(Node* n);
-  static bool is_vpopcnt_long(Node* n);
+  static bool is_downcasting_l2i_candidate(Node* n);
```

Comment:
I noticed there is  `static bool is_type_transition_short_to_int(Node* n)` above, how do you think about `is_type_transition_long_to_int`?